internal/handlers/user.go

```diff
@@ -267,45 +267,24 @@ func (c User) GetUsersList(repository domain.UserRepository) users.GetAllUsersHa
 	}
 }
 
-func (c User) DeleteUserByID(repo domain.UserRepository) users.DeleteUserHandlerFunc {
+func (c User) DeleteUserByID(repository domain.UserRepository) users.DeleteUserHandlerFunc {
```

Comment:
Can we rename 'DeleteUserByID' to 'DeleteCurrentUser' to differentiate it from user deletion by admin?